script.js

```diff
@@ -0,0 +1,139 @@
+$(document).ready(function(){
+  var num=1,
+      aleat,
+      btnClicked,
+      typeofGame,
+      allColors=['green','red','blue','yellow'],
+      rightSequence=[],
+      personSequence=[],
+      order=0,
+      storeColInf ={
+        "red": 1,
+        "green": 2,
+        "blue": 3,
+        "yellow": 4   
+      };
+  
+  changeColorDisplay('none');
+  $('.starter').attr('disabled','disabled');
```

Comment:
you are repeating this string through the code, it would be good if you set it up in a variable